client/components/domains/domain-suggestion/style.scss

```diff
@@ -251,7 +251,7 @@
 	@include break-mobile {
 		width: auto;
 		flex: 1 0 auto;
-		margin-left: 1em;
+		margin-left: 28px;
```

Comment:
This is the margin to the left of the "Select" button. By setting the pricing box to a static width and relying on the margin here for spacing, we "tighten" these two elements and prevent text-overlapping issues at small-medium widths.